src/it/scala/temple/generate/database/PostgresGeneratorIntegrationTest.scala

```diff
@@ -113,4 +113,62 @@ class PostgresGeneratorIntegrationTest extends PostgresSpec with Matchers with B
       case None => fail("Database connection could not be established")
     }
   }
+
+  "Update statements" can "update all rows in a table" in {
+    executeWithoutResults(PostgresGenerator.generate(TestData.createStatement))
+    executeWithoutResultsPrepared(PostgresGenerator.generate(TestData.insertStatement), TestData.insertDataA)
+    executeWithoutResultsPrepared(PostgresGenerator.generate(TestData.insertStatement), TestData.insertDataB)
+    executeWithoutResults(PostgresGenerator.generate(TestData.updateStatement))
+    executeWithResults("SELECT * FROM Users;") match {
```

Comment:
to avoid the repeated nesting, how about this?
```scala
val result = executeWithResults("SELECT * FROM Users;").getOrElse {
  fail("Database connection could not be established")
}
```

Basically a Swift `guard let`